src/pyodide.js

```diff
@@ -37,8 +36,10 @@ var languagePluginLoader = new Promise((resolve, reject) => {
   let _loadPackage = (names) => {
     // DFS to find all dependencies of the requested packages
     let packages = window.pyodide.packages.dependencies;
+    let loadedPackages = window.pyodide.loadedPackages;
     let queue = [].concat(names || []);
     let toLoad = new Array();
+    window.pyodide._toLoadPackages = toLoad;
```

Comment:
I think it might be cleaner to define `Module.locateFile` in this scope, then it can just access `toLoad` directly.  We'll still need to have the original `Module.locateFile` implementation at the higher scope to bootstrap the core filesystem:

```
Module.locateFile = (path) => baseURL + path;
```